iptables/iptables.go

```diff
@@ -35,6 +42,7 @@ const (
 
 var (
 	iptablesPath  string
+	iptablesVer   version
```

Comment:
The version is available in the `if iptablesPath == ` block and you can always add a new bool value.